### PR TITLE
Archer/chron job

### PR DESCRIPTION
**Pull request by ArcherHeffern**
# Github Workflow to Run API Tests

I added a github workflow to run API tests

## Changes Made

I added a github workflow to run API tests. The workflow creates a two docker containers - one running a mongodb database, and the other running the server. 
I additionally created a test file for API tests - this is what the workflow will run. 

## Reason for changes

- [x] New feature
- [ ] Bug fix
- [ ] Library upgrade(s)

### CODE REVIEW

File: .github/workflows/utils.yml [hunk 1]
```diff
@@ -46,3 +46,39 @@ jobs:
...
+            image: mongo:latest
+            ports:
+              - "27017:27017"
+    env:
+      ADMIN_PANEL_DATABASE_URL: 'mongodb://localhost:27017/testing'
+      API_TOKEN: ${{ secrets.API_TOKEN }}
+      API_TOKEN_SECRET: ${{ secrets.API_TOKEN }}
+      ADMIN_PANEL_JWT_SECRET: '17'
+      ADMIN_PANEL_SECRET: '18'
+      SAMSARA_API_KEY: 'abc'
```
Comment (sbrenner132) [inline .github/workflows/utils.yml:63]: I don't think these variables are needed. Try removing them to see if it works, but I'm pretty sure they're not needed.

File: .github/workflows/utils.yml [hunk 2]
```diff
@@ -46,3 +46,39 @@ jobs:
...
+      NEXTAUTH_URL: 'http://localhost:3000'
+      DB_ACCESS_TOKEN_SECRET: 'def'
+      UTIL: apiUtils
```
Comment (sbrenner132) [inline .github/workflows/utils.yml:66]: Same thing with this environment variable here as well.

**Review by sbrenner132 — COMMENTED** [review]
I think this is an awesome start to our API testing. There are definitely more tests we could add for the User-related routes, and we can decide on a file naming convention that would best suit our repository API documentation/testing audit script down the line, but for now I think the test written is an amazing start and proof of concept. Just take a look at the couple comments I left about the environment variables in the workflow. The fewer environment variables/cleaner code we have, the better. Just let me know once you take a look and I'll re-approve and merge!

File: .github/workflows/utils.yml (hunk 1 above)
Comment (ArcherHeffern) [reply .github/workflows/utils.yml]: do be did done

File: .github/workflows/utils.yml (hunk 2 above)
Comment (ArcherHeffern) [reply .github/workflows/utils.yml]: do  be did done

**Review by sbrenner132 — APPROVED** [review]

**Pull request merged**